docs/source/reference-core.rst

```diff
@@ -834,13 +834,14 @@ Nursery objects provide the following interface:
 
       Note that this is *not* an async function and you don't use await
       when calling it. It sets up the new task, but then returns
-      immediately, *before* it has a chance to run. The new task won’t
-      actually get a chance to do anything until some later point when
-      you execute a checkpoint and the scheduler decides to run it.
-      If you want to run a function and immediately wait for its result,
+      immediately, *before* it has a chance to run. At the next
```

Comment:
I think this is a more stringent guarantee than we actually want to make -- #32 has some discussion of scheduling models that might want us to not switch tasks at every checkpoint.

Reply:
Rewrote to not guarantee that tasks will launch at the next checkpoint.